WordPress/Classes/Utility/WPAssetExporter.m

```diff
@@ -40,7 +41,16 @@ - (void)exportAsset:(ALAsset *)asset
    stripGeoLocation:(BOOL)stripGeoLocation
   completionHandler:(void (^)(BOOL success, CGSize resultingSize, NSData *thumbnailData, NSError *error)) handler
 {
-
+    if (!asset.defaultRepresentation) {
+        if (handler) {
+            NSDictionary * userInfo = @{NSLocalizedDescriptionKey:NSLocalizedString(@"The media you are trying to use is not available locally at the moment. If it belongs to a photo stream please try again later.", @"Message that explains to a user that the current asset they selected is not available on the device. This normally happens when user selects a media that belogns to a photostream that needs to be downloaded locally first.")};
```

Comment:
It's not exactly a problem with the owner of the Photo Stream, even your Photo Streams can have this problem. Imagine you save a image on your iPad and add it to your photo stream, when you get to your iphone you can have this issue when trying to select the image.